README.md

```diff
@@ -11,6 +11,12 @@ First add the `colors` hook to your `/etc/mkinitcpio.conf` files HOOKS list. You
 will probably want to place the hook fairly eairly if you don't want the colors
```

Comment:
Could you add something above this paragraph calling out that depending on whether you're using the `systemd` vs `busybox` `initramvs`, you will have to do one of these two things.

Also maybe describe how to know which one you're using?

I actually haven't used Arch on the desktop for a while (just a couple small headless machines), is `systemd` `initramfs` becoming the default?

Reply:
1. Yes I can do that, no problem.
2. No. `busybox` is the default, when setting up Arch Linux from the `archiso`. Neither is objectively better, but rather it's down to personal preference. I use a LVM on LUKS setup and for me it is of huge convenience, that I can unlock my root LUKS device, and my data LUKS device with a single password, without the need for a `/etc/crypttab` together with encryption headers, etc. due to password caching for crypt devices in `sd-encrypt` as opposed to `encrypt` (`systemd` vs. `busybox`).

Reply:
You would know it, because as I said `busybox` is the default. Also a present `systemd` hook means it's a `systemd` based `initramfs` and `udev` is `busybox`

Reply:
But I can mention it, of course